test/test_threads.jl

```diff
@@ -4,39 +4,43 @@ using MPI
 if get(ENV,"JULIA_MPI_TEST_ARRAYTYPE","") == "CuArray"
     import CUDA
     ArrayType = CUDA.CuArray
+elseif get(ENV,"JULIA_MPI_TEST_ARRAYTYPE","") == "ROCArray"
+    import AMDGPU
+    ArrayType = AMDGPU.ROCArray
 else
     ArrayType = Array
 end
 
-provided = MPI.Init(threadlevel=:multiple)
+# DEBUG: currently failing
```

Comment:
Any intuition why these are failing? Is your local MPI not build with multiple thread support?

Reply:
No intuition on that. Maybe no multiple thread support indeed. How could I check this?

Reply:
If you are using OpenMPI use ompi_info

Reply:
```
[lraess@ault20 dev]$ ompi_info | grep "Thread support"
          Thread support: posix (MPI_THREAD_MULTIPLE: yes, OPAL support: yes, OMPI progress: no, ORTE progress: yes, Event lib: yes)
```